packages/components/src/components/ScalarIcon/icons/ArrowLeft.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="12" viewBox="0 0 14 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
what do you think about removing the width / height attributes [as highlighted here](https://github.com/scalar/scalar/tree/main/packages/components/src/components/ScalarIcon#adding-icons-to-scalaricons) in order to preserve icon responsiveness?

Reply:
oh wow I didn't see that, yea can do.

Reply:
Alright done!